apps/andi/lib/andi/harvest/harvester.ex

```diff
@@ -27,6 +33,18 @@ defmodule Andi.Harvest.Harvester do
     end
   end
 
+  def start_harvesting() do
+    case OrgStore.get_all() do
```

Comment:
This would be faster if we used a query instead. Idk how many orgs will be in the system but it's probably better to handle it now. 

If you want to keep it this way, I'm pretty sure OrgStore can return `{:ok, nil}` when there are no orgs in the system. Even though we probably won't run into that case, it would fail here on the Enum calls